src/Engine.coffee

```diff
@@ -356,7 +356,7 @@ class Engine
       unless src.match(/gss/i)
         # Select a script from document that has gss in its src
         scripts = document.querySelectorAll('script[src*=gss]')[0]
-        if scripts.length
+        if scripts?.length
```

Comment:
Once the `src` files were actually being built, this was immediately an issue when trying to run the tests. I wanted to highlight it since I'm not sure how `scripts` could be `undefined`.

Maybe it's because `runner.html` has the `<script>` include in the `head`, or that the selector matching is off.